keylime/tenant.py

```diff
@@ -936,11 +991,14 @@ def main(argv=sys.argv):
                         default=None, help="Specify a vTPM policy in JSON format")
     parser.add_argument('--verify', action='store_true', default=False,
                         help='Block on cryptographically checked key derivation confirmation from the agent once it has been provisioned')
+    parser.add_argument('--allowlist-name', help='The name of allowlist to operate with')
 
     args = parser.parse_args(argv[1:])
     mytenant = Tenant()
 
-    if args.command not in ['list', 'regdelete', 'reglist', 'delete', 'status'] and args.agent_ip is None:
+    if args.command not in ['list', 'regdelete', 'delete', 'status',
+                            'addallowlist', 'deleteallowlist',
+                            'showallowlist'] and args.agent_ip is None:
```

Comment:
Subparsers can be used to provide better experience, we can do some refactoring work  when the whole feature is implemented.

Reply:
yeah, that would be a good thing to refactor later.